core/templates/pages/exploration-editor-page/services/user-exploration-permissions.service.spec.ts

```diff
@@ -77,14 +77,22 @@ describe('User Exploration Permissions Service', () => {
   }));
 
   it('should cache rights data', fakeAsync(() => {
-    ueps.getPermissionsAsync();
+    ueps.getPermissionsAsync(false);
     let req = httpTestingController.expectOne(
       '/createhandler/permissions/' + sampleExplorationId);
     expect(req.request.method).toEqual('GET');
     req.flush(samplePermissionsData);
     flushMicrotasks();
 
-    ueps.getPermissionsAsync();
+    ueps.getPermissionsAsync(true);
+
+    let req2 = httpTestingController.expectOne(
+      '/createhandler/permissions/' + sampleExplorationId);
+    expect(req2.request.method).toEqual('GET');
+    req2.flush(samplePermissionsData);
+    flushMicrotasks();
+
+    ueps.getPermissionsAsync(false);
```

Comment:
Have seperate tests for other behaviour* (Follow the [guidelines](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests))

Reply:
DONE